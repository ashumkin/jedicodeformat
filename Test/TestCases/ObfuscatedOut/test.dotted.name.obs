unit test.dotted.name;interface implementation end. 