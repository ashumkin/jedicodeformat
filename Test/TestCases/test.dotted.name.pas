unit test.dotted.name;


{ AFS 23 May 2003
 This unit compiles but is not semantically meaningfull
 it is test cases for the code formatting utility

 This unit tests a unit name with dots in it
}

interface

implementation

end.
