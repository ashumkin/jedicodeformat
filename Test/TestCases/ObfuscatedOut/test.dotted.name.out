unit test.dotted.Name;

interface

implementation

end.
 